src/shared/ui/MovieTitle/MovieTitle.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './MovieTitle.css';
+
+export const MovieTitle = ({ imageUrl, movieName, releaseYear, genres = [], onSelect }) => {
+
+    return (
+        <div className="card">
+            <img src={process.env.PUBLIC_URL+imageUrl} alt="Avatar" />
+            <div className="container">
+                <h4><b>{movieName}</b></h4>
```

Comment:
I wouldn't recommend to use tag <b> 
<img width="759" alt="image" src="https://github.com/GaKhalatyan86/ReactLessons/assets/36797396/3d9aed8b-84c6-41e0-9ccc-99eee972c8c7">

Reply:
added css style for h4 tag